pyproject.toml

```diff
@@ -97,7 +104,8 @@ coverage = ["pytest --cov=era5cli --cov-report term --cov-report xml:cov.xml tes
 features = ["docs",]
 
 [tool.hatch.envs.docs.scripts]
-build = ["sphinx-build docs/source docs/build",]
+build = ["mkdocs build",]
```

Comment:
I get an issue:

```
peter@DESKTOP-S380217:~/ewatercycle/era5cli$ hatch run docs:build
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /home/peter/ewatercycle/era5cli/site
Traceback (most recent call last):
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/mkdocs/__main__.py", line 250, in build_command
    build.build(cfg, dirty=not clean)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/mkdocs/commands/build.py", line 297, in build
    files = config.plugins.run_event('files', files, config=config)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/mkdocs/plugins.py", line 520, in run_event
    result = method(item, **kwargs)
  File "/home/peter/.local/share/hatch/env/virtual/era5cli/XytzbWeY/docs/lib/python3.10/site-packages/mkdocs_gen_files/plugin.py", line 34, in on_files
    runpy.run_path(file_name)
  File "/usr/lib/python3.10/runpy.py", line 289, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/usr/lib/python3.10/runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/peter/ewatercycle/era5cli/docs/gen_reference_pages.py", line 77, in <module>
    helpstr = helpstr[helpstr.index("optional arguments") :]
ValueError: substring not found
```

Reply:
Oh that's weird. It works fine for me, and on ReadTheDocs as well (the built is successful). I wonder if it is a 3.10 problem, as building the docs is not in the CI.

Reply:
Perhaps your hatch environment is outdated?